README.markdown

```diff
@@ -118,6 +118,21 @@ echo($v7) # no comment here
 # My structure (Hash) {"b"=>["1", "2", "3"]}
 # (String) "12345"
 ```
+
+###cache_data
+
+Retrieves data from a cache file, or creates it with supplied data if the file doesn't exist. Useful for having data that's randomly generated once on the master side (e.g. a password), but then stays the same on subsequent runs.
+
```

Comment:
Can you document what each parameter does here? This function is a bit magic so being explicit it a good idea I think.